packages/cozy-client/src/authentication/mobile.js

```diff
@@ -84,6 +99,15 @@ const authenticateWithInAppBrowser = url => {
   })
 }
 
+/**
+ * Open a RactNative InAppBrowsr
```

Comment:
There are some typos:

Opens a ReactNative InAppBrowser and resolves with the URL
containing the token